src/start_testbuilds.py

```diff
@@ -94,14 +118,18 @@ def main():
             pr_test_job = f"dlc-pr-{test_type}-test"
             images_str = " ".join(images)
             if is_test_job_enabled(test_type):
-                if "huggingface" in images_str and test_type in [constants.EC2_TESTS, constants.ECS_TESTS, constants.EKS_TESTS]:
+                if "huggingface" in images_str and test_type in [
+                    constants.EC2_TESTS,
+                    constants.ECS_TESTS,
+                    constants.EKS_TESTS,
+                ]:
```

Comment:
nit: Strange that this got broken into multiple lines. Is the line-length limit set to 120 characters?

Reply:
yep, ran w/ -l 120

Reply:
you can do a copy paste on the line length, it's 128